trusted-committer/05-lowering-the-barriers-to-entry.asciidoc

```diff
@@ -1,75 +1,62 @@
+_shared repository_ and _fork and join_. Both have advantages, and as a Trusted Committer,
+you want to support both models to accommodate different needs of your
 potential and current https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_].
-Oftentimes, potential https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] will have questions they would like to
-have answered before they make a contribution. Those could be tech oriented
-questions, questions regarding contributions, or quite simply questions aimed at
-figuring out if there's somebody to talk to in the community. It is therefore
-important for any InnerSource community to have one or more contact persons
-that are available for answering such questions. It is the Trusted Committer's responsibility
-to make sure there is a community member "on call". Most commonly, the Trusted Committers
-themselves will fulfill that role, since onboarding new community members is
-one of their jobs.
-
-It is also important to help potential https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_]
-to determine what contributions are welcome or needed. These can be code
-contributions but also non-code contributions, such as writing documentation,
-creating artwork or organizing events. One common way to do this is to tag
-"newcomer tasks" in the issue tracker used by the community or implement a
-marketplace for open tasks, that https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] can use.
-
-
-In summary, it is super important for InnerSource communities in a corporate
-environment to keep the barriers to contributing as low as possible to get as
-many https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] as possible. Trusted Committers therefore make sure that users and
-https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] have both access to helpful documentation and people in the
-community to answer any questions they might have and that they can get up
-and running in no time. Generally speaking, Trusted Committers should make sure that the
-the onboarding experience is a great one.
+Your Contributors will often have questions about the contribution process or about the community itself, and someone has to be available to answer these questions. 
+It is therefore important for any InnerSource community to
+have one or more contact persons that are available for answering such
+questions. The Trusted Committer is usually that contact person, or else they need to make sure there is a community member "on call".
```

Comment:
This sounds to me like there would only be one Trusted Committer, however there can be multiple. 
Putting up a suggestion here.
```suggestion
questions. Someone from the group of Trusted Committers is usually that contact person, or else they need to make sure there is a community member "on call".
```